documentation/book/con-certificate-renewal.adoc

```diff
@@ -0,0 +1,31 @@
+[id='con-certificate-renewal-{context}']
+= Certificate Renewal
+
+CA certificates (_cluster CA_ and _clients CA_) are only valid for given number of days after they are generated.
+When using auto-generated CA certificates this time period defaults to 365 days.
+Manually installed CA certificates will come with their own validity period.
+
+If a CA certificate has expired then certificates signed by it will not be valid, even if they are still within their own period of validity. This usually means that new client certificates must also be generated when a CA certificate is going to expire.
```

Comment:
Are you saying that when cluster cas expire you need new client cas? or are you only talking about client cas here? It's not clear.

Reply:
How about:

When a CA certificate expires certificates signed by it will fail validation, even if they are otherwise valid. This means when a CA certificate is replaced it is also necessary to replace all the certificates previously signed by that CA.

Reply:
Sounds good to me!